groupmne/inverse.py

```diff
@@ -7,93 +7,356 @@
+                                                subjects)):
+        if hemi == "lh":
+            v = [v_l, []]
+        elif hemi == "rh":
+            v = [[], v_r]
+        else:
+            v = [v_l, v_r]
+        stc = utils._make_stc(coefs[:, :, i].T, v, tmin=group_info["tmin"],
+                              tstep=group_info["tstep"], subject=subject)
+        stcs.append(stc)
+    return stcs
+
+
+def _method_to_solver(method):
+    if method == "lasso":
+        return IndLasso
+    elif method == "relasso":
+        return IndRewLasso
+    elif method == "mtw":
+        return MTW
+    elif method == "remtw":
+        return ReMTW
+    elif method == "dirty":
+        return DirtyModel
+    elif method == "grouplasso":
+        return GroupLasso
+    else:
+        raise ValueError("Method %s not recognized." % method)
+
+
+def _method_to_str(method):
+    if method == "lasso":
+        return "mutar.IndLasso"
+    if method == "relasso":
+        return "mutar.IndRewLasso"
+    elif method == "mtw":
+        return "mutar.MTW"
+    elif method == "remtw":
+        return "mutar.ReMTW"
+    elif method == "dirty":
+        return "mutar.DirtyModel"
+    elif method == "grouplasso":
+        return "mutar.GroupLasso"
+    else:
+        raise ValueError("Method %s not recognized." % method)
+
+
+class InverseOperator(object):
+    """InverseOperator class to represent the inverse problem data.
 
     Parameters
     ----------
-    gains: ndarray, shape (n_subjects, n_channels, n_sources).
-        Forward data, returned by
-        `group_model.compute_gains` or `group_model.compute_inv_data`.
-    M : ndarray, shape (n_subjects, n_channels, n_times)
-        M-EEG data
-    group_info : dict
-        The measurement info.
-    method : str
-        Inverse problem model to use. For now, only "grouplasso" is
-        supported. The group-lasso solver promotes source estimates with
-        overlapping active vertices across subjects. Each time point is
-        treated independently.
-    depth : float in (0, 1)
-        Depth weighting. If 1, no normalization is done.
-    alpha : float in (0, 1)
-        Regularization hyperparameter set as a fraction of
-        alpha_max for which all sources are 0.
-    return_stc : bool, (optional, default True)
-        If true, source estimates are returned as stc objects, array otherwise.
-    time_independent : bool, (optional, default false)
-        If True, each time point is solved independently. By default,
-        the group lasso is applied on the time and subjects axes.
-    n_jobs : int (default 1)
-        The number of CPUs to use in parallel.
-    kwargs : dict
-        additional arguments passed to the solver.
-
-    Returns
-    -------
-    estimates: instance of SourceEstimates | array
-        The estimated sources as an array if `return_stc` is False.
-    log : dict
-        Some info about the convergence.
+    fwds: list of `mne.Forward`.
```

Comment:
```suggestion
    fwds: list of `mne.Forward`
```
remove all the dots for parameter types. Just needed to short description below.